websites/K/Kickstarter/presence.ts

```diff
@@ -0,0 +1,77 @@
+const presence = new Presence({ clientId: "1014441192106229790" }),
+	browsingTimestamp = Math.floor(Date.now() / 1000),
+	slideshow = presence.createSlideshow(),
+	staticsTabsProject: { [name: string]: string } = {
+		description: "Reading project description",
+		faqs: "Reading project FAQ",
+		posts: "Reading project news",
+		comments: "Reading project comments",
+		community: "Viewing project contributors",
+		pledge: "Contributing to the project",
```

Comment:
You can use Title Case here

Reply:
Resolved in 1fdbed86bf8344a15b1a6430d2b4aa244e35ee9f